x-pack/plugins/apm/server/plugin.ts

```diff
@@ -248,11 +253,6 @@ export class APMPlugin
       config: this.currentConfig,
       logger: this.logger,
     });
-
-    migrateLegacyAPMIndicesToSpaceAware({
```

Comment:
was this moved somewhere else? is it no longer necessary?

Reply:
thanks, i think this is a result of a bad merge